### DISCUSSION TITLE
Problem with writing characters with diacritical marks

### DISCUSSION THREAD

**Discussion by VaTi1** [rustdesk/rustdesk#920] [General]
Hello. When working on a remote computer (both local and remote computers have Windows 10 with the same keyboard layout and language settings) it is not possible to type characters with specific diacritical marks (e.g. **ď**, **Ď**, **ť**, **Ť**, **ň**, **Ň**, etc. in Czech language), which are typed with key combinations (see https://en.wikipedia.org/wiki/Dead_key) on a Czech keyboard (QWERTZ or QWERTY). 
For example, the letter "**ň**" is typed by simultaneously pressing the "**SHIFT**" and "**ˇ**" keys (the key to the left of the Backspace key) and then pressing the "**n**" key. Is there any solution to this problem? Thanks

![diacritical_marks](https://user-images.githubusercontent.com/108670387/177153417-d75324fd-91e5-4795-96e1-a06eda76493e.jpg)
.

**Comment by rustdesk** [comment]
@asur4s

**Comment by rustdesk** [comment]
Thanks

**Comment by jernejs** [comment]
I [reported](https://github.com/rustdesk/rustdesk/issues/406#issuecomment-1133765731) something similar with Slovenian keyboard – in my case, (most) dead keys work fine in remote session, but stop working locally while RustDesk is running.

**Reply by rustdesk** [reply]
Thanks

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/964
